content/paths/task_assignments__put_task_assignments_id.yml

```diff
@@ -29,8 +29,10 @@ requestBody:
           resolution_state:
             type: string
             description: |-
-              The state of the task. This can be used to
-              mark the task as completed.
+              The state of the task. This can be completed or incomplete for
```

Comment:
I'd like to see all values wrapper in backticks, e.g "`completed` or `incomplete`, as it makes it clear to the translation team not to translate these as they are fixed values.